### ISSUE TITLE
Tabbar icons resize mode?

### ISSUE THREAD

**Issue by Estebank94** [wix/react-native-navigation#5173]
### Issue Description

Hello,

I am having trouble displaying icons that have a bigger width than height in Android.
I am assuming the problem is because the default image resize mode in Android is cover so part of the icon looks cropped.

How can I set the resize mode of the icons? If it's not possible, which is a possible solution?

### Steps to Reproduce / Code Snippets / Screenshots

![Screenshot_1559357630](https://user-images.githubusercontent.com/5116089/58773976-c009cd80-8595-11e9-9cd3-d8c24ffa4556.png)

To reproduce the problem, add an icon that has a bigger width than height to the tabbar. For example if you are using FontAwesome: shopping-cart or truck.

---
### Environment
* React Native Navigation version: 1.1.381
* React Native version: 0.55.2
* Platform: Android
* Device info: Emultaor

**Comment by OrLavy** [comment]
I also suffer from this problem. 

These kind of problems make it really hard to use this library, we obviously cannot ship a product when our bottom tabs in Android are all cropped out.

**Comment by Estebank94** [comment]
I am still having this problem. Did anyone find a solution?

**Comment by OrLavy** [comment]
I also suffer from this problem, this thread should definitely not be declared stale.

**Comment by lukebrandonfarrell** [comment]
Still a problem on Android, any solutions ?

**Comment by lukebrandonfarrell** [comment]
"react-native-navigation": "^2.21.0",
"react-native": "^0.59.9",

Still an issue, any solutions @OrLavy @Estebank94 ?

@guyca Could we re-open this issue?

I drilled down into what may be causing this issue and think it could be fixed if we forked [ahbottomnavigation](https://github.com/aurelhubert/ahbottomnavigation) and set a ScaleType to `CENTER_INSIDE` (or make this customisable?) in `AHBottomNavigation` at Line 530 using `setScaleType` on the `ImageView`.

I think this may be an issue in the 'ahbottomnavigation' library, not `react-native-navigation`, but still unsure.
